libnavigation-ui/src/main/java/com/mapbox/navigation/ui/feedback/FeedbackBottomSheet.java

```diff
@@ -57,52 +54,46 @@
   private static final String EMPTY_FEEDBACK_DESCRIPTION = "";
   private static final long CLOSE_BOTTOM_SHEET_AFTER = 150L;
   private static final long TIMER_INTERVAL = 1L;
-  private static final int GRID_SPAN_GUIDANCE_LAYOUT = 3;
-  private static final int GRID_SPAN_NAVIGATION_LAYOUT = 3;
+  private static final int GRID_SPAN_PORTRAIT = 3;
+  private static final int GRID_SPAN_LANDSCAPE = 6;
 
   @Nullable
   private FeedbackBottomSheetListener feedbackBottomSheetListener;
   private TextView feedbackBottomSheetTitleText;
   private ImageButton cancelBtn;
-  private LinearLayout feedbackMainLayout;
-  private RecyclerView guidanceIssueItems;
-  private FeedbackAdapter guidanceIssueAdapter;
-  private RecyclerView navigationIssueItems;
-  private FeedbackAdapter notificationIssueAdapter;
-  private RelativeLayout feedbackSubTypesLayout;
+  private RecyclerView feedbackCategories;
+  private FeedbackAdapter feedbackCategoryAdapter;
   @Nullable
-  private FeedbackSubTypeAdapter feedbackSubTypeAdapter;
-  private RecyclerView feedbackSubTypeItems;
   private ProgressBar feedbackProgressBar;
   private ObjectAnimator countdownAnimation;
-  private AppCompatButton reportIssueBtn;
-  private @FeedbackFlowType int feedbackFlowType = FEEDBACK_MAIN_FLOW;
   private long duration;
   @Nullable
   private CountDownTimer timer = null;
-  private FeedbackItem selectedFeedbackItem;
-  private Map<String, List<FeedbackSubTypeItem>> feedbackSubTypeMap;
-  private Class<? extends FeedbackBottomSheetListener> listenerClass;
+  private Class<? extends FeedbackBottomSheetListener> feedbackBottomSheetListenerClass;
   @Nullable
   private DismissCommand dismissCommand = null;
 
   @NonNull
   public static FeedbackBottomSheet newInstance(FeedbackBottomSheetListener feedbackBottomSheetListener,
                                                 long duration) {
-    return newInstance(feedbackBottomSheetListener, FEEDBACK_MAIN_FLOW, duration);
-  }
-
-  @NonNull
```

Comment:
This is breaking SemVer we should put it back (even if we're adding `@Deprecated`).